### ISSUE TITLE
Can't switch profiles to ODBC (MySQL) - Error in your SQL syntax

### ISSUE THREAD

**Issue by nikkadim** [signalwire/freeswitch#2058]
Can't switch profiles to ODBC (MySQL)

```
2023-04-21 21:55:46.103647 97.07% [ERR] switch_odbc.c:529 ERR: [delete from sip_registrations where sub_host is null and hostname='pfs' and network_ip like '%' and network_port like '%' and sip_username like '%' and mwi_user  like '%' and mwi_host like '%' and orig_server_host like '%' and orig_hostname like '%';delete from sip_registrations where sub_host is null and hostname='pfs' and network_ip like '%' and network_port like '%' and sip_username like '%' and mwi_user  like '%' and mwi_host like '%' and orig_server_host like '%' and orig_hostname like '%']
[STATE: 42000 CODE 1064 ERROR: [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.35-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'delete from sip_registrations where sub_host is null and hostname='pfs' and netw' at line 1
]
2023-04-21 21:55:46.103647 97.07% [ERR] switch_core_sqldb.c:728 [db="pfs_core";type="odbc"user="fs_is_the_best";pass="******"] ODBC SQL ERR [STATE: 42000 CODE 1064 ERROR: [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.35-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'delete from sip_registrations where sub_host is null and hostname='pfs' and netw' at line 1
]
delete from sip_registrations where sub_host is null and hostname='pfs' and network_ip like '%' and network_port like '%' and sip_username like '%' and mwi_user  like '%' and mwi_host like '%' and orig_server_host like '%' and orig_hostname like '%';delete from sip_registrations where sub_host is null and hostname='pfs' and network_ip like '%' and network_port like '%' and sip_username like '%' and mwi_user  like '%' and mwi_host like '%' and orig_server_host like '%' and orig_hostname like '%'
2023-04-21 21:55:46.103647 97.07% [CRIT] sofia_glue.c:2679 GREAT SCOTT!!! Cannot execute batched statements! [STATE: 42000 CODE 1064 ERROR: [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.35-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'delete from sip_registrations where sub_host is null and hostname='pfs' and netw' at line 1
]
If you are using mysql, make sure you are using MYODBC 3.51.18 or higher and enable FLAG_MULTI_STATEMENTS
2023-04-21 21:55:46.103647 97.07% [CRIT] sofia.c:3159 Cannot Open SQL Database [internal_phones]!
```

**To Reproduce**
1. Adding to the profile internal_phones.xml:
```
    <param name="odbc-dsn" value="odbc://pfs_core:fs_is_the_best:**********"/>
    <param name="track-calls" value="true"/>
```
2. sofia profile internal_phones restart 
3. and getting errors above

- FreeSWITCH (Version 1.10.9 -release-21-a615e85afc 64bit) 
- Debian GNU/Linux 11 (bullseye)
- MySQL 5.7.35
- ODBC 8.0(a) Driver

 FLAG_MULTI_STATEMENTS is enabled by OPTION=67108864
```
[pfs_core]
Driver = /usr/lib/x86_64-linux-gnu/odbc/libmyodbc8a.so
SERVER  = 10.1.1.1
PORT     = 3306
DATABASE = pfs_core
USER = fs_is_the_best
PASSWORD = **********
OPTION = 67108864

```

PS: core-db  switched to odbc via _switch.conf.xml_ `<param name="core-db-dsn" value="odbc://pfs_core:fs_is_the_best:*************"/>` and works fine!

**Comment by nikkadim** [comment]
resolved downgrading to odbc-5.3.13

**Comment by andywolk** [comment]
Why do you use ODBC when there is mod_mariadb available?
https://github.com/signalwire/freeswitch/blob/master/conf/vanilla/autoload_configs/pre_load_modules.conf.xml#L4
https://github.com/signalwire/freeswitch/blob/master/conf/vanilla/autoload_configs/switch.conf.xml#L183
